robots/flakefinder/report.go

```diff
@@ -0,0 +1,322 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"html/template"
+	"log"
+	"os"
+	"path"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/joshdk/go-junit"
+)
+
+const tpl = `
+<!DOCTYPE html>
+<html>
+<head>
+<title>kubevirt.io - flakefinder report</title>
+<meta charset="UTF-8">
+<style>
+table, th, td {
+  border: 1px solid black;
+}
+.yellow {
+  background-color: #ffff80;
+}
+.almostgreen {
+  background-color: #dfff80;
+}
+.green {
+  background-color: #9fff80;
+}
+.red {
+  background-color: #ff8080;
+}
+.orange {
+  background-color: #ffbf80;
+}
+.unimportant {
+}
+.center {
+  text-align:center
+}
+
+/* Popup container - can be anything you want */
+.popup {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+/* The actual popup */
+.popup .popuptext {
+  visibility: hidden;
+  width: 220px;
+  background-color: #555;
+  text-align: center;
+  border-radius: 6px;
+  padding: 8px 8px;
+  position: absolute;
+  z-index: 1;
+  bottom: 125%;
+  left: 50%;
+  margin-left: -110px;
+}
+
+.nowrap {
+  white-space: nowrap;
+}
+
+/* Popup arrow */
+.popup .popuptext::after {
+  content: "";
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  margin-left: -5px;
+  border-width: 5px;
+  border-style: solid;
+  border-color: #555 transparent transparent transparent;
+}
+
+/* Toggle this class - hide and show the popup */
+.popup .show {
+  visibility: visible;
+  -webkit-animation: fadeIn 1s;
+  animation: fadeIn 1s;
+}
+
+/* Add animation (fade in the popup) */
+@-webkit-keyframes fadeIn {
+  from {opacity: 0;} 
+  to {opacity: 1;}
+}
+
+@keyframes fadeIn {
+  from {opacity: 0;}
+  to {opacity:1 ;}
+}
+</style>
+</head>
+<body>
+
+<table>
```

Comment:
could you format that HTML? It's quite hard to read.

Reply:
Done.